### ISSUE TITLE
Screen reader accessibility

### ISSUE THREAD

**Issue by ZettaScript** [marchellodev/sharik#167]
The Linux desktop app does not seem accessible using the screen reader Orca.

Usually a UI can be made (mostly) accessible by adding labels and roles to elements, and ensuring every element can be focused using tab or arrows.

I know this is possible (on Linux) with GTK, but I don't know about Flutter. However GTK is usually not accessible on Windows (this can be tested using NVDA). Flutter should be accessible on Android though (can be tested using TalkBack).

Edit: A friend told me it's accessible on Android, but not on Windows (it says only "FLUTTERVIEW").
